src/courtroom.cpp

```diff
@@ -322,6 +322,8 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
   connect(ui_effects_dropdown, SIGNAL(customContextMenuRequested(QPoint)), this,
           SLOT(on_effects_context_menu_requested(QPoint)));
 
+  connect(ui_music_search, SIGNAL(returnPressed()), this,
+          SLOT(on_music_search_keypr()));
```

Comment:
on_music_search_keypr should be renamed to on_music_search_return_pressed like before, and its signal needs to be added to header